service/model/release.go

```diff
@@ -37,7 +37,7 @@ type Release struct {
 	GitTagURL    url.URL
 }
 
-func NewRelease(input CreateReleaseInput, projectID, authorUserID uuid.UUID) (Release, error) {
+func NewRelease(input CreateReleaseInput, gitTagURL url.URL, projectID, authorUserID uuid.UUID) (Release, error) {
```

Comment:
Here I'd definitely expect only `CreateReleaseInput`.

I think something like `NewCreateReleaseInput` function (somewhere near `Release` model) would probably the cleanest way. 🤔